portfolio/src/main/webapp/index.html

```diff
@@ -174,8 +174,43 @@ <h4>
         <div class="info">     
           <h2>Projects</h2>
 
-          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-          </p>
+          <br>
+          <br>
+
+          <div class="container">
+            <div class="card">
+              <div class="imgBx">
+                <img src=images/motm.jpg width="200" height="175">
+              </div>
+              <div class="content">
+                <h2>Word2Vec Retrieval Induced Forgetting</h2>
+                <p>Investigate word2vec machine learning models to learn salient stimuli in retrieval induced forgetting and conducting online study testing memory inhibition.</p>
+              </div>
+            </div>
+
+            <div class="card">
+              <div class="imgBx">
+                <img src=images/mindfulnest.png width="200" height="175">
+              </div>
+              <div class="content">
+                <h2>CREATE Lab MindfulNest</h2>
+                <p> Design front end interface for Android mobile app, MindfulNest, teaching pre-K students emotion control and self-calming with interactive breathing and stress relieving exercises through the app interface.
+                </p>
+              </div>
+            </div>
+
+            <div class="card">
+              <div class="imgBx">
+                <img src=images/ai.jpg width="200" height="175">
+              </div>
+              <div class="content">
+                <h2>Augmented Reality Futuring Project</h2>
```

Comment:
Using h2 here makes it the same hierarchy as the Projects header, maybe you want h3?